backend/cron.js

```diff
@@ -0,0 +1,85 @@
+const CronJob = require('cron').CronJob
+const PullRequest = require('./services/PullRequest')
+const redisClient = require('./redis')
+
+let startDate = '2019-10-01 00:00:00'
+const endDate = '2019-10-31 23:59:59'
+const year = new Date(startDate).getFullYear()
+
+async function runCrons () {
+  const updatePullRequests = new CronJob('*/5 * * * *', async () => {
+    try {
+      let latestTimestampInRedis = await redisClient.get('latest_timestamp')
+
+      if (latestTimestampInRedis) {
+        startDate = latestTimestampInRedis
+      }
+
+      let pullRequest = new PullRequest(startDate, endDate)
+
+      let response = await pullRequest.getAll()
+
+      let latestTimestamp = response.latestTimestamp()
+      await redisClient.set('latest_timestamp', latestTimestamp)
+
+      response
+        .groupByUser()
+        .sortByMostActive()
+
+      let data = response.data
+      let prsPromises = []
```

Comment:
`getAll()` returns the instance, like other chainable methos from  `PullRequest` so something like this should work.
I'm nort 100% positive due to use of async/await, but you got the idea.

```js
const pullRequestsList = await new PullRequest(startDate, endDate)
  .getAll()
  .groupByUser()
  .sortByMostActive()
  .data // a getData() method would be better

  await redisClient.set('latest_timestamp', response.latestTimestamp())
```

Ans also avoid variable names like `response` or `data`, unless their use is very specific in the context (e.g. response from a request) which is not the case here.